.coveragerc

```diff
@@ -10,6 +10,7 @@ omit =
     # omit pieces of code that rely on external devices being present
     homeassistant/components/acer_projector/switch.py
     homeassistant/components/actiontec/device_tracker.py
+    homeassistant/components/acmeda/*
```

Comment:
We still want to track `config_flow.py` so please leave that in and add entries for the files that are missing tests.

Reply:
Done.